### PR TITLE
Docker Build

### PR DESCRIPTION
**Pull request by Emersont1**

### CODE REVIEW

File: src/hstp.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Metadata-Version: 2.1
```
Comment (jb3) [inline src/hstp.egg-info/PKG-INFO:1]: egg-info should be excluded from CI

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import setuptools
+
+with open("Readme.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
```
Comment (jb3) [inline setup.py:6]: We could use entrypoints here to install HSTP as a proper command line tool

https://packaging.python.org/en/latest/specifications/entry-points/

**Review by jb3 — CHANGES_REQUESTED** [review]
Couple of comments, otherwise all lgtm

File: src/hstp.egg-info/PKG-INFO (hunk 1 above)
Comment (Emersont1) [reply src/hstp.egg-info/PKG-INFO:1]: Cool, I'll delete and gitignore

**Pull request merged**